trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py

```diff
@@ -59,28 +59,28 @@ def __defaults__(self):
             Properties Used:
             N/A
         """             
-        self.motor_lift                = None
-        self.motor_forward             = None
-        self.rotor                     = None
-        self.propeller                 = None
-        self.esc_lift                  = None
-        self.esc_forward               = None
-        self.avionics                  = None
-        self.payload                   = None
-        self.battery                   = None
-        self.nacelle_diameter_lift     = None
```

Comment:
This doesn't have to do with this PR specifically. But are these diameters meant to be used for drag? In that case I don't think they'd be read:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_propulsor.py#L57